crates/core/src/db/message_log.rs

```diff
@@ -195,6 +195,14 @@ impl std::fmt::Debug for MessageLog {
     }
 }
 
+impl Drop for MessageLog {
+    fn drop(&mut self) {
+        if let Err(e) = self.sync_all() {
```

Comment:
Do we need to do the same with the objectDB?

Reply:
Maybe -- but sled will try to flush its pagecache on drop already, so it's unclear if it would buy us much.